docs/Wallet.md

```diff
@@ -118,6 +120,18 @@ The Receive tab generates an unused address which can be used to receive payment
 
 ![Wallet Receive](./img/wallet/WalletReceive.png)
 
+### Pull Payments
+
+This feature gives you the ability to create a Pull Payment, so that an outside individual may `pull` funds frop his wallet.
```

Comment:
```suggestion
This feature gives you the ability to create a Pull Payment, so that an outside individual may request to `pull` funds from your wallet.
```